website/docs/d/cm_catalog.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "ibm"
+page_title: "IBM : cm_catalog"
+sidebar_current: "docs-ibm-datasource-cm-catalog"
```

Comment:
1. Add you service name in https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/allowed-subcategories.txt
2. remove sidebar_current in all your docs
3. Add sub category at the beginning soomething like..
`subcategory: "Catalog Management`
Refer other docs for reference